ProjectLighthouse.Servers.API/Responses/RpcResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿#nullable disable
+
+using LBPUnion.ProjectLighthouse.Configuration.ConfigurationCategories;
+
+namespace LBPUnion.ProjectLighthouse.Servers.API.Responses;
+
+public class RpcResponse
```

Comment:
The whole RPC JSON schema should be documented somewhere so that anyone who wants to implement it can, it might be worth writing a small description of this and putting it in /Documentation

Reply:
So that things like RPC can be standard within the LBP custom server community, even for those who don't want to use PLRPC for instance. Without any developers having to dig through the Lighthouse source code

Reply:
Can do